### DISCUSSION TITLE
Request URL - unauthorized error

### DISCUSSION THREAD

**Discussion by annmc** [amberelectric/public-api#13] [Q&A] [answered]
I am trying to run the **Request URL**s from https://app.amber.com.au/developers/ on the Chrome browser, but they all return the error **{"message":"Unauthorized"}**.

This is with the authorize value set to my token ID. Do I need to change this, perhaps, when trying to run the URLs? or maybe add a parameter to the URL?
Thanks
Ann

**Comment by madpilot** [comment, marked as answer]
They won't work if you paste them in to the browser URL bar - they require the bearer token as part of the Authorization header.

Is that what you were trying to do?

**Reply by annmc** [reply]
Ah, ok. Yes, that's what I was trying to do. Very new to all this. Working my way towards integrating this with google home, one miniscule piece of knowledge at a time. 

Thanks for your help today. The api and the documentation look great.
